### PR TITLE
update sponsorship package

### LINKED ISSUE #2: Add recurring sponsorship tiers

### PR DESCRIPTION
**Pull request by philweier**
A simplified sponsorship package with rates based on duration.
closes #2

### CODE REVIEW

**Comment by jenncoop** [comment]
This is a bit hard to read:

screen shot 2016-09-29 at 7 25 24 am

I think you meant it to be a list, no?

Otherwise, looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-- Thanked during intro by organizer
-- Can say a few words (up to 1 minute max)
-- Option of 1 speaker spot by staff member for the event
-- Can put up company banner etc.
+Sponsorship rates are (We run 12 events per year):
+Month to Month $250
+6 Months $1300
+12 Months $2400
+12 Month Exclusive Sponsor $4000
+
```
Comment (whilestevego) [inline README.md:21]: I agree with @jenncoop that this part should be a list 😃

**Review by whilestevego — CHANGES_REQUESTED** [review]
Aside from that one change. It looks great. Thanks for the update!

**Review by whilestevego — APPROVED** [review]

**Pull request merged**